test-bdd/templates/rolling-upgrade.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: upgrademgr.keikoproj.io/v1alpha1
```

Comment:
This template is not consistent with the new CRD. 
You have to make changes by looking into the CRD in upgrade-manager-v2

https://github.com/keikoproj/upgrade-manager/blob/controller-v2/config/crd/bases/upgrademgr.keikoproj.io_rollingupgrades.yaml